### DISCUSSION TITLE
FTDI board

### DISCUSSION THREAD

**Discussion by PhilPhal** [ctu-mrs/mrs_uav_system#67] [Q&A] [answered]
Hi MRS team,

During the construction of a second f450 drone, I come across some problems with the communication between the Nuc and the PixHawk. The "Rx" blue light of the FTDI board doesn't light up when I connect it. I disassembled my first f450 drone to make sure I did the same and I can't find any differences. I also tried placing the FTDI board and cables on the first drone, and there the connection succeeds.

Do you have an idea what could be the issue?

Thank you in advance.
Philippe

**Comment by spurnvoj** [comment, marked as answer]
Hi, check the pixhawk sd card - https://ctu-mrs.github.io/docs/hardware/px4_configuration.html#sd-card-setup.

**Comment by PhilPhal** [comment]
Hi,

Thank you for the quick response. I was unaware of that setup... It fixed the problem!

Have a nice day,
Philippe
